package.json

```diff
@@ -26,18 +27,18 @@
   "author": "Shannon Moeller <me@shannonmoeller.com> (http://shannonmoeller.com)",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/shannonmoeller/svgstore/issues"
+    "url": "https://github.com/svgstore/svgstore/issues"
   },
-  "homepage": "https://github.com/shannonmoeller/svgstore",
+  "homepage": "https://github.com/svgstore/svgstore",
   "dependencies": {
-    "cheerio": "^0.20.0",
+    "cheerio": "^0.22.0",
     "object-assign": "^4.1.0"
   },
   "devDependencies": {
-    "ava": "0.15.2",
+    "ava": "0.16.0",
```

Comment:
I think it will be okay here, but would be even better if we had individual PRs for tracking dependency updates. I sent a request for enabling [Greenkeeper](https://greenkeeper.io/) access to the svgstore org, which would issue beautiful automated PRs ([like this one for AVA](https://github.com/declandewet/common-tags/pull/50)) every time a dep gets upgraded.